react/AppLinker/index.jsx

```diff
@@ -49,12 +53,20 @@ export class AppLinker extends React.Component {
     }
   }
 
-  static getOnClickHref(props, nativeAppIsAvailable) {
+  getOnClickHref(props, nativeAppIsAvailable) {
     const { slug, nativePath } = props
     let href = props.href
     let onClick = null
     const usingNativeApp = isMobileApp()
     const appInfo = NATIVE_APP_INFOS[slug]
+
+    if (isFlagshipApp()) {
+      return {
+        onClick: () => this.context.call('openWebview', href),
```

Comment:
we should check if this.context is defined when isFlagShip is true. If not, console.warn() and then we do as today we don't call context.call